support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -24,19 +35,20 @@ case class EmailFields(
   fields: List[(String, String)],
   userId: Either[SfContactId, IdentityUserId],
   email: String,
-  dataExtensionName: String
+  dataExtensionName: String,
+  deliveryDate: Option[LocalDate] = None,
 ) {
 
-  def payload: String = {
+  def payload: String =
     EmailPayload(
       To = EmailPayloadTo(
         Address = email,
         ContactAttributes = EmailPayloadContactAttributes(SubscriberAttributes = fields.toMap)
       ),
       DataExtensionName = dataExtensionName,
       SfContactId = userId.left.toOption.map(_.id),
-      IdentityUserId = userId.right.toOption.map(_.id)
-    ).asJson.spaces2
-  }
+      IdentityUserId = userId.right.toOption.map(_.id),
+      deliveryDate.map(_.toDateTime(new LocalTime(8, 0), DateTimeZone.UTC)),
+    ).asJson.printWith(Printer.spaces2.copy(dropNullValues = true))
```

Comment:
dropNullValues isn't the default but we use it in some places to prevent None becoming null.  For the payload to membership-workflow it just reduces the clutter.